llvm/test/Feature/Repo/repo_hash_graph_dependencies_1.ll

```diff
@@ -0,0 +1,26 @@
+; Test GO's information of Dependencies - global variable.
```

Comment:
The first line comment is the same as the previous file. Could we describe how they differ?

Reply:
Changed in commit cca1f2d. The comments describe the difference between these two tests.